manifests/crd/v1/pingcap.com_dataresources.yaml

```diff
@@ -0,0 +1,46 @@
+
+---
```

Comment:
No need to generate crd for this.

Reply:
DataResource is not used?

Reply:
No.

Reply:
It seems that this crd exists in `manifests/data-resource-crd.yaml`